test/kubernetes/e2e/features/upstreams/suite.go

```diff
@@ -0,0 +1,67 @@
+package upstreams
+
+import (
+	"context"
+
+	"github.com/onsi/gomega"
+	"github.com/onsi/gomega/gstruct"
+	"github.com/solo-io/solo-kit/pkg/api/v1/clients"
+	"github.com/solo-io/solo-kit/pkg/api/v1/resources"
+	"github.com/solo-io/solo-kit/pkg/api/v1/resources/core"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/solo-io/gloo/pkg/utils/kubeutils"
+	"github.com/solo-io/gloo/pkg/utils/requestutils/curl"
+	"github.com/solo-io/gloo/test/kubernetes/e2e"
+)
+
+// testingSuite is the entire Suite of tests for the "Route Options" feature
+type testingSuite struct {
+	suite.Suite
+
+	ctx context.Context
+
+	// testInstallation contains all the metadata/utilities necessary to execute a series of tests
+	// against an installation of Gloo Gateway
+	testInstallation *e2e.TestInstallation
+}
+
+func NewTestingSuite(ctx context.Context, testInst *e2e.TestInstallation) suite.TestingSuite {
+	return &testingSuite{
+		ctx:              ctx,
+		testInstallation: testInst,
+	}
+}
+
+func (s *testingSuite) TestConfigureBackingDestinationsWithUpstream() {
+	s.T().Cleanup(func() {
+		err := s.testInstallation.Actions.Kubectl().DeleteFile(s.ctx, routeWithUpstreamManifest)
+		s.NoError(err, "can delete manifest")
+		s.testInstallation.Assertions.EventuallyObjectsNotExist(s.ctx, proxyService, proxyDeployment)
+	})
+
+	err := s.testInstallation.Actions.Kubectl().ApplyFile(s.ctx, routeWithUpstreamManifest)
+	s.Assert().NoError(err, "can route to gloo.solo.io Upstreams")
```

Comment:
This is just asserting the resources are applied, right? 

```suggestion
	s.Assert().NoError(err, "can apply gloo.solo.io Upstreams manifest")
```